arangod/MMFiles/MMFilesRestReplicationHandler.cpp

```diff
@@ -360,7 +362,13 @@ void MMFilesRestReplicationHandler::handleCommandLoggerFollow() {
     }
   }
 
-  ExecContextSuperuserScope escope(ExecContext::current().isAdminUser());
```

Comment:
@graetzer @jsteemann what is protected by this? The _users collection? The _wal? Why is a new scope needed?